src/sources/kafka.rs

```diff
@@ -389,7 +396,7 @@ mod test {
     #[tokio::test]
     async fn consumer_create_ok() {
         let config = make_config("topic", "group");
-        assert!(create_consumer(&config).is_ok());
+        assert!(create_consumer(&config, true).is_ok());
```

Comment:
Do we have any tests with autocommit false?

Reply:
Not to just create the base consumer here. We do have integration tests with and without acknowledgements.